MergePythonSDK/accounting/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
same deal not needed inside a category folder, along with `setup.py`, `test-requirements.txt` and `tox.ini`